makefile

```
@@ -0,0 +1,13 @@
+DOCKER_COMPOSE_YML = docker/apache.yml
+
+build:
```

I would add `.PHONY` declaration for `build`, `stop`, `start` and `clean`, because those recipes aren't existing files:
```suggestion
.PHONY: build stop start clean

build:
```